app/routers/organizations.py

```diff
@@ -1,15 +1,43 @@
 from fastapi import APIRouter, status, HTTPException
 from database import client
 from models import Organization
+from fastapi.responses import JSONResponse
+import string
+import random
 
 router = APIRouter()
 
 
+@router.post("/create-org/{org_name}")
+async def create_organization(org_name: str):
+    if org_name is not None:
+        # create an API key
+        char_set = string.ascii_letters + string.punctuation
+        urand = random.SystemRandom()
+        org_key = "".join([urand.choice(char_set) for _ in range(20)])
```

Comment:
What is this line doing? Is this what is used in Plio to generate keys, @deepansh96 ?

Reply:
In Plio, they used the secrets library, but essentially the same thing.

Reply:
Use the same library and a small loop as mentioned below